src/praisonai-agents/praisonaiagents/tools/jira_tools.py

```
@@ -0,0 +1,377 @@
+        
+        result = f"Found {len(issues)} issues for JQL: {jql}\n\n"
+        
+        for issue in issues:
+            result += f"{issue.key}: {issue.fields.summary}\n"
+            result += f"  Status: {issue.fields.status.name}\n"
+            result += f"  Assignee: {issue.fields.assignee.displayName if issue.fields.assignee else 'Unassigned'}\n"
+            result += f"  Updated: {issue.fields.updated}\n\n"
+        
+        return result
+        
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)